assignments/function-conversion.js

```diff
@@ -1,23 +1,37 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 
+let anotherFunction = param => param;
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
-let subtract = function (param1, param2) {
-  return param1 - param2;
-};
-subtract(1,2); //?
+let add = (param1, param2) => {return param1 + param2};
```

Comment:
This format works :), but be careful for when you have to make a longer function. Good start to the assignment 👍